CRM/Contact/Selector.php

```diff
@@ -283,24 +283,25 @@ public static function &links() {
           'url' => 'civicrm/contact/map',
           'qs' => "reset=1&cid=%%id%%{$searchContext}{$extraParams}",
           'title' => ts('Map Contact'),
+          'weight' => 10,
```

Comment:
Should we set this globally so that `CRM_Core_Action::getWeight(CRM_Core_Action::MAP)` returns `10`?

Reply:
@colemanw is it called from anywhere else? I just picked a random number :-)

Reply:
Don't think so. Your random number is as good as any.